### ISSUE TITLE
Wireless thermostats not showing in Homebridge

### ISSUE THREAD

**Issue by johnny11fingers** [henry-spanka/homebridge-freeathome#35]
Hi,
First of all I'm stunned with the job you did merging free@home with Apple HomeKit. I do admire what you did.
Yesterday I configured my Homebridge with free@home PlugIn for the first time. My ABB system manages all shutters and thermostats and some of the lights. I successfully went through configuration process and can see now lights and shutters in iOS Home app. I'm missing all thermostats, both in Homebridge UI and thus, in iOS Home.
Thermostats are correctly configured in free@home and I can manage them through native application.
Is it possible to manually add missing accessories?
Thanks in advance for your help,
Jan

**Comment by henry-spanka** [comment]
Hi,
glad that you like my work. Please send me an export of the project from your System Access Point. You can send it to my email address shown on my GitHub profile.

**Comment by uwe-21** [comment]
Hi guys,
really great stuff this plugin! And many thanks to henry-spanka!!

A lot of my devices are already showing up in Homebridge, except wireless thermostats.
It seems that I have exactly the same issue than Jan above. 
Is there already a solution to get wireless Busch Jäger thermostats into homebridge?

Thanks,
Uwe

**Comment by henry-spanka** [comment]
Is this still relevant?

**Comment by johnny11fingers** [comment]
Hi, 

I stopped using your app a while ago, need to buy raspberry as it was running on my Mac air which made no sense. Still thanks to you raspberry is in my expense pipeline. 

I’ll keep you posted once I’m back home and able to test. 

Thanks for doing this, you’re addressing a huge gap with ***@***.***

Cheers,
J. 

Jan Polakowski

**Comment by nixmeer** [comment]
Hello :)

I got the same issue: all of my 10 thermostats are not showing up neither on the web UI nor in the home app. However, the plugin seems to detect them, since I get the following message 10x after starting homebridge:
`[15/05/2022, 11:42:32] [homebridge-freeathome] This plugin generated a warning from the characteristic 'Target Temperature': characteristic was supplied illegal value: number 0 exceeded minimum of 7. See https://homebridge.io/w/JtMGR for more info.`

Maybe that helps?

**Comment by Glampi** [comment]
> Is this still relevant?

Yes, thermometers do not show up for me as well, and I couldn't find anything online besides this [Busch Jäger community post ](https://community.busch-jaeger.de/frage-ansehen/lokale-rest-api-mit-der-homebridge-freeathome-benutzen-falls-jemand-interesse-hat) mentioning setting `"isLocalAPI": true `. That didn't work for me.

Edit: Nevermind, that community post is about a fork of your plugin, it looks like. But, the thermometers do not show up for me still.
